catalog/plugin/catalog-plugin-oauth/src/main/java/org/codice/ddf/catalog/plugin/oauth/OAuthPlugin.java

```diff
@@ -140,17 +141,22 @@ public QueryRequest process(Source source, QueryRequest input) throws StopProces
 
     Object securityAssertion = input.getProperties().get(SECURITY_SUBJECT);
     if (!(securityAssertion instanceof Subject)) {
-      LOGGER.warn("A user Subject is not available.");
-      throw new StopProcessingException("A user Subject is not available.");
+      LOGGER.warn("The user's subject is not available.");
+      throw new StopProcessingException("The user's subject is not available.");
     }
 
     Subject subject = (Subject) securityAssertion;
-    String userId = SubjectUtils.getEmailAddress(subject);
-    if (userId == null) {
-      userId = SubjectUtils.getName(subject);
+    Session session = subject.getSession();
```

Comment:
❓ Looks like this will create a session by default. Is that the intent? https://shiro.apache.org/static/1.4.1/apidocs/org/apache/shiro/subject/support/DelegatingSubject.html#getSession-boolean-

Reply:
No, I'll change that